custom_components/bambu_lab/pybambu/models.py

```diff
@@ -125,6 +130,38 @@ def TurnChamberLightOff(self):
         self.client.publish(CHAMBER_LIGHT_OFF)
 
 
+@dataclass
+class Camera:
+    """Return camera related info"""
+    recording: str
+    resolution: str
+    rtsp_url: str
+    timelapse: str
+
+    def __init__(self):
+        self.recording = ''
+        self.resolution = ''
+        self.rtsp_url = None
+        self.timelapse = ''
+
+    def print_update(self, data):
+        """Update from dict"""
+        # "ipcam": {
+        #   "ipcam_dev": "1",
+        #   "ipcam_record": "enable",
+        #   "mode_bits": 2,
+        #   "resolution": "1080p",
+        #   "rtsp_url": "rtsps://192.168.1.64/streaming/live/1",
```

Comment:
Not something to worry about for this PR but we might be able to populate the camera availability based on whether this exists in the mqtt payload instead of gating it on the model. But probably more work than benefit.